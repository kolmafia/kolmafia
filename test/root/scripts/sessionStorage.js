const { print, sessionStorage } = require("kolmafia");

sessionStorage.setItem("control", "wow");

sessionStorage.setItem("test", "a");
sessionStorage.removeItem("test");
print(`Remove: ${sessionStorage.getItem("test") === null}`);

print(`Control: ${sessionStorage.getItem("control") === "wow"}`);

sessionStorage.setItem("test", "a");
sessionStorage.clear();
print(`Clear: ${sessionStorage.getItem("test") === null}`);