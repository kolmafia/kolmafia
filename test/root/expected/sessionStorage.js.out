Remove: true
Control: true
Clear: true
